tests/unit-tests/test-validate-post-install.sh

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env bash
 
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
 source src/validate-post-install.sh
```

Comment:
We are inconsistent in our expectation of where these tests should be launched from...
Options:
  1- from the top most directory, like 'test-validate-post-install.sh' is setup
  2- from src/tests/unit-tests directory, like 'test-cmd-parser.sh'
  3- from src/tests, like perf-tests